packages/manager/apps/dedicated/client/app/components/dedicated-cloud/license/enable/dedicatedCloud-license-enable.constants.js

```diff
@@ -0,0 +1,11 @@
+export const OVERRIDE_CONTRACTS = [
+  {
+    NAME: 'Conditions Particulières Private Cloud Windows',
+    TYPE: 'MBM',
+    URL: 'https://www.ovh.com/fr/support/documents_legaux/EULA_Microsoft.pdf',
+  },
+];
+
+export default {
+  OVERRIDE_CONTRACTS,
```

Comment:
shouldn't those be returned by an API ? (like `/order/cart/checkout` )

Reply:
Yes... But in this case, the API team said they will not be able to do it. Hence, we had to handle this in the UI. Had discussed this with @frenautvh too. Seems like we have no other option as of now.